lib/travis/api/app/endpoint/builds.rb

```diff
@@ -66,5 +66,11 @@ class Builds < Endpoint
         respond_with(result: result, flash: service.messages)
       end
     end
+
+    private def include_log_id?
+      params[:include_log_id] ||
```

Comment:
No, this is one that I added in order to deal with the very different request/traffic patterns of web and travis.rb.  I've included it as a param check, too, mostly for verification and testing purposes.